src/main/resources/assets/goodgenerator/data/essentia.json

```diff
@@ -152,19 +152,19 @@
     },
     {
       "name": "Praecantatio",
-      "fuelValue": 36000,
+      "fuelValue": 30000,
       "category": "NORMAL",
       "consumeCeo": 0.0
     },
     {
       "name": "Auram",
-      "fuelValue": 42000,
+      "fuelValue": 76800,
       "category": "AIR",
-      "consumeCeo": 2.7
+      "consumeCeo": 7.0
     },
     {
       "name": "Vitium",
-      "fuelValue": 4700,
+      "fuelValue": 4200,
```

Comment:
This doesn't fix the issue much, I mean it will make an impact but like 🤷

Reply:
Refer to the chart above. vitium's actual benefits have been greatly reduced, it now generates less than a tenth of what it used to generate